filecoin-proofs/src/api/seal.rs

```diff
@@ -81,8 +82,18 @@ pub fn seal_pre_commit<R: AsRef<Path>, T: AsRef<Path>, S: AsRef<Path>>(
         _,
     >>::setup(&compound_setup_params)?;
 
+    // MT for original data is always named tree-d, and it will be
+    // referenced later in the process as such.
+    let cache_dir = cache_path.as_ref().to_str().unwrap();
+    let config = StoreConfig::new(
+        cache_dir.to_string(),
```

Comment:
this should really take a `T: AsRef<Path>`, allowing us to just pass the argument through

Reply:
I like the idea here, but might need some pointers on how to do this properly.  Everything broke in merkle_light regarding the configs with my first attempt ... but will definitely keep in mind as a needed clean-up.

Reply:
lets connect on Monday when you are up